src/api/app/models/package.rb

```diff
@@ -939,7 +944,7 @@ def buildresults(prj = project)
       end
     end
 
-    local_build_results
+    return local_build_results, excluded_number
```

Comment:
I don't really like this approach :cry: And I also think that it might be better to move this function to a dedicated model now.

What about something like this:
https://gist.github.com/ChrisBr/4e7de315410ea8ccf8cdc149eb210d3e
(just quick and dirty, highly untested. Just to get my idea :smile: )

tl;dr: it would allow sth like:
```
local_build_results = LocalBuildResult::ForPackage.new(package)
local_build_results.all
local_build_results.count(excluded: true)
local_build_results.count
```

It would also make testing easier.

We can brainstorm on that together tomorrow ...